pkg/controller/storagecluster/initialization_reconciler.go

```diff
@@ -121,19 +157,13 @@ func (r *ReconcileStorageCluster) ensureCephObjectStores(instance *ocsv1.Storage
 	if err != nil {
 		return err
 	}
-	if platform == PlatformAWS {
-		r.reqLogger.Info("not creating a CephObjectStore because the platform is AWS")
-		return nil
-	}
-	if platform == PlatformGCP {
-		r.reqLogger.Info("not creating a CephObjectStore because the platform is GCP")
-		return nil
-	}
-	if platform == PlatformAzure {
-		r.reqLogger.Info("not creating a CephObjectStore because the platform is Azure")
+	if r.isCloudPlatform(platform) {
+		r.reqLogger.Info("Cloud Platform: '" + string(platform) +
+			"', detected. Not creating CephObjectStore.")
```

Comment:
```suggestion
		r.reqLogger.Infof("Cloud Platform: %q, detected. Not creating CephObjectStore.", string(platform))
```

Doesn't `reqLogger` have an `Infof`?

Reply:
Nope... It only has `Info()` method for logging... Or I could use `fmt.Sprintf()` inside the `Info()` method...

Reply:
😨 ok...